src/main/java/com/example/customerdb/controller/PurchaseInfoController.java

```diff
@@ -34,11 +34,17 @@ public void addInfo(@RequestBody PurchaseInfo purchaseInfo) {
         purchaseInfoService.addInfo(purchaseInfo);
     }
 
+
     @PutMapping("/purchase-info/{id}")
     public void updateInfo(@PathVariable int id, @RequestBody PurchaseInfo purchaseInfo) {
         purchaseInfoService.updateInfo(id, purchaseInfo);
     }
 
+    @PatchMapping("/purchase_info/{id}")
```

Comment:
たぶんですが、
```suggestion
    @PatchMapping("/purchase-info/{id}")
```
アンダースコアでなくハイフンではないでしょうか？
ハイフンに直せば405も解消すると思います。

Reply:
修正したところ直りました！
動作確認を投げさせていただきます。
![スクリーンショット 2023-06-30 11 45 23](https://github.com/Yumu0112/Task10/assets/118417799/e985bf5a-5894-40d7-88d0-341846230d47)

Reply:
13c0ec7460497ed96e4344a2ebcbec9b925b7a

レスポンスの追加を行いました！

![スクリーンショット 2023-06-30 13 19 13](https://github.com/Yumu0112/Task10/assets/118417799/6bb88763-c1cb-4d6e-ba41-e68e45e12819)

Reply:
32bd073bb1ed39422f1242c897b4d062d9655f0f

少し冗長な書き方をしてしまいましたが、とりあえずJSONレスポンスの返却を行いました。

![スクリーンショット 2023-06-30 13 54 43](https://github.com/Yumu0112/Task10/assets/118417799/cbdb9726-ae84-4b00-9f40-fca43fab4c13)

Reply:
d02b7b690f4a079f4b669199a5d5d3e865a46a61

修正版